profile/README.md

```diff
@@ -5,3 +5,104 @@
     \__, /_/ /_/\____/      Social:    <a href="https://twitter.com/_gnoland">Twitter</a> - <a href="https://discord.gg/tF2X8M6cVj">Discord</a> - <a href="https://reddit.com/r/gnoland">Reddit</a> - <a href="https://youtube.com/@_gnoland">YouTube</a>
    /____/                   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 </pre>
+
+***
+
+## Why Gno? 
+
+- Interpreted Gnolang: Gno.land utilizes the Gnolang programming language, which is based on Go. It is executed through a specialized virtual machine called the GnoVM, purpose-built for the Gno.land blockchain. While Gnolang shares similarities with Go in terms of syntax, it currently lacks go routine support. However, this feature is planned for future development, ensuring deterministic GnoVM executions.
+
+- Consensus Protocol - Tendermint2: Gno.land achieves consensus between blockchain nodes using the Tendermint2 consensus protocol. This approach ensures secure and reliable network operation.
```

Comment:
Okay, as a rule of thumb, use the question "Why is this better than ethereum / bitcoin?"

```suggestion
- Consensus Protocol: Gno.land achieves consensus between nodes using the Tendermint2 consensus algorithm. Tendermint2 is a variation on the well-established [Tendermint consensus algorithm](https://docs.tendermint.com/v0.34/introduction/what-is-tendermint.html) used in many Cosmos blockchains, which is Byzantine Fault Tolerant, has instant finality (Users can be sure their transactions are finalized as soon as a block is created, unlike Bitcoin) and can easily be extracted to create new interoperable TM2-based blockchains.
```